src/pages/notes/board/MoveableText.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {Path, Skia, Text} from '@shopify/react-native-skia';
+import {GenericMovableElement} from "./GenericMovableElement";
+import {svgEditButton, svgRectangleBorder} from "./Utils";
+
+interface MovableTextProps {
+    font: any;
+    element: GenericMovableElement;
+}
+
+const MovableText = ({
+      font, element
+}: MovableTextProps) => {
+
+    return (
+        <>
+            {(element.isEditingMode || element.isMoving) && <Path
+                path={Skia.Path.MakeFromSVGString(svgRectangleBorder(
+                    element.position.x - 5, element.position.y, element.content.length * 10, element.height))!}
```

Comment:
why -5 and *10?

Reply:
Currently there is no option to change the font size of the text and `10` is static value for one single letter. So we are multipling the length of the word times `10`. The `-5` is for drawing the rectangle border in the some radius from the text. (btw I do not know why we don't have to add `1` to content.length value but it works as expected xD)